users/service.go

```diff
@@ -577,7 +583,7 @@ func (svc service) OAuthCallback(ctx context.Context, state mgoauth2.State, clie
 			if errors.Contains(err, repoerr.ErrConflict) {
 				return &magistrala.Token{}, errors.New("user already exists")
```

Comment:
Wrapping  this error with `svcerr.ErrConflict` will return conflict status code in HTTP response